packages/mobile/src/onboarding/registration/NameAndNumber.tsx

```diff
@@ -77,7 +77,8 @@ function getPhoneNumberState(
 
   if (phoneDetails) {
     return {
-      nationalPhoneNumber: phoneDetails.displayNumber,
+      // Show international display number to avoid confusion
+      nationalPhoneNumber: phoneDetails.displayNumberInternational,
```

Comment:
I propose we also rename the field to further avoid confusion:
```suggestion
      internationalPhoneNumber: phoneDetails.displayNumberInternational,
```